pmd-jsp/src/main/java/net/sourceforge/pmd/lang/jsp/ast/AbstractJspNode.java

```diff
@@ -6,7 +6,7 @@
 
 import net.sourceforge.pmd.lang.ast.AbstractNode;
 
-public class AbstractJspNode extends AbstractNode implements JspNode {
+public abstract class AbstractJspNode extends AbstractNode implements JspNode {
```

Comment:
adding this `abstract` is a breaking API change. Even though I agree it makes perfect sense to have it, we can't break this API until PMD 7.0.0. We should revert this changeset. We could make a reminder ticket like we did with #463 with 6.0.0

Reply:
Done, it's in #881